docs/guides/admin/performance_monitoring.md

```diff
@@ -0,0 +1,106 @@
+# Performance Monitoring
+
+Timesketch includes a built-in performance profiler to help diagnose and troubleshoot performance issues. When enabled, it captures detailed profiling data for every HTTP request to the backend and saves it for later analysis.
+
+## Enabling the Profiler
+
+To enable the performance profiler, you need to edit your `timesketch.conf` file and set the `DEBUG` configuration option to `True`.
```

Comment:
Do you want to add a line in the timesketch.conf as well that mentions this will happen when DEBUG set to true?